Shopify/cli
# Small GraphiQL fixes

### WHY are these changes introduced?

Fixes a few items from the bug bash.



### WHAT is this pull request doing?



1. Adds a minimum 20px horizontal margin on small screens on the unauthorized screen

    Before:

    <img width="521" alt="Screenshot 2023-11-29 at 23 08 42" src="https://github.com/Shopify/cli/assets/6288426/93a40e00-a6af-47ad-8e84-0552aa25e482">

    After:
    <img width="519" alt="Screenshot 2023-11-29 at 23 02 15" src="https://github.com/Shopify/cli/assets/6288426/417dcb0b-0f35-4297-b3a1-0091fd9563e3">

2. Ensures link pills have proper nowrap behavior cross-browser (previously it worked on Chrome but not Safari)

    Before:

    <img width="223" alt="Screenshot 2023-11-29 at 23 09 32" src="https://github.com/Shopify/cli/assets/6288426/a199a5bf-9718-4fe8-ba26-a2d5fd206095">

    After:
    
    <img width="220" alt="Screenshot 2023-11-29 at 23 06 58" src="https://github.com/Shopify/cli/assets/6288426/16ce2c90-c5a2-40af-9925-7e2acc94aee7">

4. Changes the status pill to warn that the app was uninstalled if it was uninstalled while GraphiQL is running

    Before:

    <img width="146" alt="Screenshot 2023-11-29 at 23 10 12" src="https://github.com/Shopify/cli/assets/6288426/55737893-f0bb-4c4b-b895-71e859bb818c">

    After:

    <img width="227" alt="Screenshot 2023-11-29 at 23 07 33" src="https://github.com/Shopify/cli/assets/6288426/e3a724c6-a488-4314-8a2b-e2479667cd0c">

5. GraphiQL favicon!

    <img width="264" alt="Screenshot 2023-11-29 at 23 15 15" src="https://github.com/Shopify/cli/assets/6288426/6cd4336c-6352-4887-9e4a-d2ce46acfc06">

### How to test your changes?



1. Start with an uninstalled app. Spin up `dev` and tap `g` to open GraphiQL.
2. On the unauthorized screen, use devtools to look at a very small screen (<480px width). You should still have a horizontal margin.
3. Install the app.
4. Copy the GraphiQL link and open in Safari. Ensure the link pills' text stays on one line.
5. Without closing GraphiQL, uninstall the app in another tab. Within a few seconds, you should see the green `Running` status pill turn to a yellow `App uninstalled` status pill. If you install again, it should go back to green.
6. Confirm your GraphiQL tab's favicon looks like the GraphQL logo

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
